nominatim/tools/check_database.py

```diff
@@ -146,11 +148,46 @@ def check_connection(conn: Any, config: Configuration) -> CheckResult:
 
     return CheckState.OK
 
+@_check(hint="""\
+             Database version ({db_version}) doesn't match Nominatim version ({nom_version})
+
+             Hints:
+             * Are you connecting to the correct database?
+             
+             A) If database version is lower than Nominatim version: Run migrations
+                nominatim admin --migrate
+ 
+             B) If Nominatim version is lower than database version: Update Nominatim
```

Comment:
Hand in the appropriate hint in a variable, then you don't have to confuse people with the two possibilities.

Reply:
addednow